src/pages/dmt5-trading-signals/index.js

```diff
@@ -45,7 +45,7 @@ const signal_content_provider = {
         <Localize translate_text="If you are a professional trader, the MT5 trading signals service allows you to share your strategies with other traders for free or a subscription fee that you determine. When traders subscribe to your signal, your deals are automatically replicated on their accounts each time you place a trade." />
     ),
     list: [
-        <Localize translate_text="Easy to set up – no installation required" key={0} />,
+        <Localize translate_text="Easy to set up – no installation required." key={0} />,
```

Comment:
Based on the design, these 3 should not have a fullstop actually

Reply:
The design team already confirmed it needs to have a fullstop

https://redmine.deriv.cloud/issues/7250